forui/lib/src/widgets/select_group/select_group.dart

```diff
@@ -38,43 +40,53 @@ class FSelectGroup<T> extends StatelessWidget {
   final List<FSelectGroupItem<T>> items;
 
   /// Creates a [FSelectGroup].
-  const FSelectGroup({
+  FSelectGroup({
     required this.controller,
     required this.items,
     this.style,
     this.label,
     this.description,
     this.error,
+    super.onSaved,
+    super.validator,
+    super.initialValue,
```

Comment:
How does this work with a controller?

Reply:
* `onSaved` is called when the values are saved, no direct interaction with the controller.
* `validator` is called when the fields are validated, no direct interaction with the controller.
* `initialValue` is used when the `reset()` method is called and when the state's internal value is set to null.

In most cases, the state and controller are synced via `_handleControllerChanged` and `didChange(...)` methods.